Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/UserInput.kt

```diff
@@ -450,7 +450,7 @@ fun EmojiSelector(
     Column(
         modifier = Modifier
             .focusRequester(focusRequester) // Requests focus when the Emoji selector is displayed
-            .focus() // Make the emoji selector focusable so it can steal focus from TextField
```

Comment:
why was "emoji" removed? it's important

Reply:
Because it didn't fit in one line anymore. But let's add it back again and have the comment in a different line